src/main/java/graphql/nadel/engine/QueryTransformationResult.java

```diff
@@ -89,5 +93,7 @@ public OperationDefinition getOperationDefinition() {
     public TransformationMetadata getRemovedFieldMap() {
         return removedFieldMap;
     }
+
+    public List<String> getHintTypenameMap() { return hintTypenameMappings; }
```

Comment:
The naming here is all over the place - I think you need to decide on a map or list and reflect that in naming.

I havent got to the place where its consumed but its already a bit skew if

Reply:
I am confused here - this list (or map) is never used in code  - is this a setup for some other code not in this PR?

One thing to consider is introducing a class that represents this list / map of things.   Sure its justa  list / map under the covers but if its called `TypeNameMappings` then 1) you can find all usages more easily and 2) methods that take it as a parameter will be a smidge clear - they take NOT just a map / list but a named thing

Reply:
found the read part above